src/android.ts

```diff
@@ -2,12 +2,12 @@ import { decompress as decompressXz } from '@napi-rs/lzma/xz';
 import { runAndroidDevTool } from 'andromatic';
```

Comment:
In the example script, the app never gets started for me. Subsequently, `getPrefs()` fails:

```
file:///home/benni/coding/JS/tweasel/appstraction/node_modules/andromatic/node_modules/execa/lib/error.js:59
		error = new Error(message);
		        ^

Error: Command failed with exit code 1: /home/benni/.cache/andromatic/platform-tools/adb shell pidof -s de.hafas.android.db
    at makeError (file:///home/benni/coding/JS/tweasel/appstraction/node_modules/andromatic/node_modules/execa/lib/error.js:59:11)
    at handlePromise (file:///home/benni/coding/JS/tweasel/appstraction/node_modules/andromatic/node_modules/execa/index.js:124:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at Object.getPidForAppId (/home/benni/coding/JS/tweasel/appstraction/src/android.ts:604:28)
    at Object.getPrefs (/home/benni/coding/JS/tweasel/appstraction/src/android.ts:610:21)
    at <anonymous> (/home/benni/coding/JS/tweasel/appstraction/examples/android-device.ts:52:19) {
  shortMessage: 'Command failed with exit code 1: /home/benni/.cache/andromatic/platform-tools/adb shell pidof -s de.hafas.android.db',
  command: '/home/benni/.cache/andromatic/platform-tools/adb shell pidof -s de.hafas.android.db',
  escapedCommand: '"/home/benni/.cache/andromatic/platform-tools/adb" shell pidof -s de.hafas.android.db',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: '',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

Reply:
This does happen for me as well with the DB Navigator, but not with other apps. Did you test with the DB Navigator?

Reply:
I did use DBN, yes.

Reply:
Actually, my problem was that my version of DB Navigator is a split APK, but the example only works for single APKs. Single APKs all work fine really.